webapp/components/signup/components/signup_email.jsx

```diff
@@ -40,7 +40,7 @@ export default class SignupEmail extends React.Component {
         this.state = this.getInviteInfo();
     }
 
-    componentDidMount() {
+    static componentDidMount() {
```

Comment:
Removed this in https://github.com/mattermost/platform/pull/7014/commits/34b87fe88a7e1c05ce97324861124e531a8a840b